ick/cmdline.py

```diff
@@ -111,17 +113,35 @@ def run(ctx, dry_run: bool, patch: bool, yolo: bool, filters: list[str]):
 
     # DO THE NEEDFUL
 
+    rule_modified = {}
```

Comment:
Since this has more information than just whether files were modified, a better name might be `results`